BATTLES.md

```diff
@@ -0,0 +1,66 @@
+# Battles
+
+A guide to battles.
+
+## Starting Battles
+
+The code below immediately initiates a battle with a level 1 Rattata.
+It can be used before selecting "continue" after loading the FullScreenPokemon index.html file.
+This is useful when repeatedly testing minor edits to battle animations.
+
+```
+var pokemonTitle = "RATTATA";
+var pokemon = FSP.utilities.createPokemon({
+    title: "RATTATA".split(""),
+});
+
+FSP.battles.startBattle({
+    battlers: {
+        opponent: {
+            actors: [pokemon],
+            name: pokemonTitle,
+            category: "Wild",
+            sprite: pokemonTitle + "Front"
+        }
+    }
+});
+```
+
+When making short edits like timing/positioning changes, the Cutscenes.js file can be directly modified.
+This avoids the use of "gulp" after editing the .ts file, which can take a while.
+The .js file can either be modified with a text editor, or with the Chrome DevTools.
+
+
+## Animation Additions
+
+To create new animations, the function must be added in src/Cutscenes.ts with the following format:
+
+```
+public cutsceneBattleAttackAttackName(settings: IBattleCutsceneSettings, args: IBattleAttackRoutineSettings): void {
+    ...
```

Comment:
Can you mention the behavioral traits common to all battle cutscenes? For example, direction?